models.py

```diff
@@ -18,6 +18,8 @@ class Book(Base):
     id: Mapped[int] = mapped_column(primary_key=True, index=True)
     title: Mapped[str] = mapped_column(String(255), index=True)
     author: Mapped[str] = mapped_column(String(255))
+    desc: Mapped[str] = mapped_column(String(255), nullable=False)
```

Comment:
```suggestion
```suggestion
    desc: Mapped[str] = mapped_column(String(255), nullable=False)  # existing field from the database model
```
```